packages/mgt-components/src/components/mgt-agenda/mgt-agenda.scss

```
@@ -9,198 +9,267 @@
+    opacity: 0.9;
+  }
 
-:host .agenda-list,
-mgt-agenda .agenda-list {
-  list-style-type: none;
-  padding: 0;
-  margin: 0;
-  font-family: $font-family;
-  font-style: normal;
-  font-weight: normal;
-}
+  .agenda,
+  .group {
+    display: flex;
+    flex-direction: column;
+    row-gap: var(--event-row-gap, 14px);
 
-:host .event,
-mgt-agenda .event {
-  background: set-var(event__background-color, $theme-default, $agenda);
-  border: var(--event-border, solid 2px rgba(0, 0, 0, 0));
-  box-shadow: var(--event-box-shadow, 0px 2px 8px set-var(box-shadow__color, $theme-default, $common));
-  margin: var(--event-margin, 0px 10px 14px 10px);
-  padding: var(--event-padding, 0px);
-  display: flex;
-  border-radius: 8px;
-}
+    & > .group:first-child > .header {
+      margin-top: 0;
+    }
 
-:host .narrow .event,
-mgt-agenda .narrow .event {
-  flex-direction: column;
-}
+    .event {
+      background: set-var(event__background-color, $theme-default, $agenda);
+      border: var(--event-border, solid 2px rgba(0, 0, 0, 0));
+      box-shadow: var(--elevation-shadow-card-rest, --event-box-shadow);
+      padding: var(--event-padding, 12px);
+      position: relative;
+      display: flex;
+      flex: 1 1 auto;
+      content-visibility: visible;
+      contain: none;
 
-:host .event-time-container,
-mgt-agenda .event-time-container {
-  display: flex;
-  flex-direction: column;
-  margin: 18px 38px 18px 18px;
-  width: 125px;
-}
+      &-container {
+        border-radius: calc(var(--layer-corner-radius) * 1px);
+        padding: 1px;
+      }
 
-:host .narrow .event-time-container,
-mgt-agenda .narrow .event-time-container {
-  margin: 14px 18px 8px 18px;
-}
+      &.narrow {
+        display: flex;
+        flex-direction: column;
+        inset: 0px;
+      }
 
-:host .event-time,
-mgt-agenda .event-time {
-  font-size: var(--event-time-font-size, 12px);
-  color: $event__time__color;
-  font-weight: 600;
-}
+      &-time-container {
+        font-family: var(--default-font-family);
+        font-style: normal;
+        font-weight: 600;
+        font-size: 12px;
+        color: $event__time__color;
+        width: 112px;
+        height: 16px;
 
-:host .event-duration,
-mgt-agenda .event-duration {
-  color: rgba(16, 16, 16, 0.3);
-}
+        &.narrow {
+          margin-bottom: 1px;
+          width: 100%;
+        }
+        .event {
+          &-time-loading {
+            @include loading-element;
+          }
+        }
+      }
 
-:host .event-details-container,
-mgt-agenda .event-details-container {
-  margin: 14px 18px 14px 0px;
-}
+      &-time {
+        font-size: var(--event-time-font-size, 12px);
+        color: $event__time__color;
+        font-weight: 600;
+      }
 
-:host .narrow .event-details-container,
-mgt-agenda .narrow .event-details-container {
-  margin: 0px 18px 14px 18px;
-}
+      &-details-container {
+        position: relative;
+        bottom: 8px;
+        top: 0px;
+        left: 32px;
```

make this a margin-left and the horizontal scrollbar that was just introduced is eliminated.
```suggestion
        margin-left: 32px;
```

---

I tried this and it's didn't work. I found another sneaky one, setting the `--card-width: 99%`